sklearn/preprocessing/tests/test_data.py

```diff
@@ -1651,6 +1651,17 @@ def test_robust_scaler_zero_variance_features():
     assert_array_almost_equal(X_trans_new, X_expected_new, decimal=3)
 
 
+def test_robust_scaler_unit_variance():
+    # Check RobustScaler with unit_variance=True is equivalent to
+    # StandardScaler on large normal data
+    rng = np.random.RandomState(0)
+    X = rng.randn(100000, 1) * 5 + 2
+    robust_scaler = RobustScaler(unit_variance=True)
+    standard_scaler = StandardScaler()
+    assert_allclose(robust_scaler.fit_transform(X),
+                    standard_scaler.fit_transform(X), rtol=20)
```

Comment:
ouch a relative tolerance of 20 seems that we will never catch any failure.
I will check what we can do.

Reply:
I know! We need a really large random sample. Or to test this differently...?